index.mjs

```diff
@@ -103,7 +103,11 @@ export async function question(query, options) {
     output: process.stdout,
     completer,
   })
-  const question = promisify(rl.question).bind(rl)
+  let question = promisify(rl.question).bind(rl)
+  if (!rl.question[promisify.custom]) {
```

Comment:
What about promisifying ourselves instead of relaying on util.promisify gere?

Reply:
Sure, you mean to say we do this right ? https://github.com/google/zx/pull/37/commits/b47e8ea2e07b2e6a761081fc43f5d2b281a43cdc